From c739bf214b9dd6060db216b79077806fccb582ae Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex@linutronix.de>
Date: Fri, 17 Nov 2023 14:26:32 +0100
Subject: [PATCH] Lib/sysconfig.py: use prefix value from build configuration
 file

This allows correctly substituting them for target installs using
native python.

Upstream-Status: Inappropriate [oe-core cross builds]
Signed-off-by: Alexander Kanavin <alex@linutronix.de>
---
 Lib/sysconfig.py | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/Lib/sysconfig.py b/Lib/sysconfig.py
index 6258b68..d59ec6e 100644
--- a/Lib/sysconfig.py
+++ b/Lib/sysconfig.py
@@ -675,6 +675,11 @@ def _init_config_vars():
         _CONFIG_VARS['VPATH'] = sys._vpath
     if os.name == 'posix':
         _init_posix(_CONFIG_VARS)
+    _CONFIG_VARS['installed_base'] = _CONFIG_VARS['prefix']
+    _CONFIG_VARS['base'] = _CONFIG_VARS['prefix']
+    _CONFIG_VARS['installed_platbase'] = _CONFIG_VARS['prefix']
+    _CONFIG_VARS['platbase'] = _CONFIG_VARS['prefix']
+    _CONFIG_VARS['platlibdir'] = _CONFIG_VARS['PLATLIBDIR']
     if _HAS_USER_BASE:
         # Setting 'userbase' is done below the call to the
         # init function to enable using 'get_config_var' in
